sensor_msgs/CompressedImage image
DetectedFace[] detected_faces